aniso8601/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Many of your recipe PRs include this.  I think you're using it to get rid of the egg?  That's fine, but on Windows, I have never gotten ```--single-version-externally-managed``` to work without the ```--root=c:\``` argument also being provided.  I generally use ```--old-and-unmanageable``` instead, which seems to behave equivalently to the two-argument solution, just with a slightly unnerving name.

If you have tested this on Windows, I'll take your word for it and merge it.